src/robot/running/builder/transformers.py

```diff
@@ -229,6 +229,11 @@ def visit_ReturnStatement(self, node):
         self.test.body.create_keyword(name='RETURN', args=node.values,
                                       lineno=node.lineno)
 
+    def visit_Continue(self, node):
+        self.test.body.create_keyword(name='CONTINUE',lineno=node.lineno)
+
+    def visit_Break(self, node):
+        self.test.body.create_keyword(name='BREAK',lineno=node.lineno)
 
 class KeywordBuilder(NodeVisitor):
```

Comment:
1. When mapping CONTINUE and BREAK used outside a loop to a keyword call, we should pass also their possible arguments. That's done with RETURN above, but using `node.values` isn't possible here because CONTINUE and BREAK nodes don't currently have such attribute. Probably `NoArgumentsHeader` should get this property to allow that:
   ```python
    @property
    def values(self):
        return self.get_values(Token.ARGUMENT)
    ```
2. There should be two empty lines between classes.

Reply:
In this case, I am adding back args=node.values to the method. 

def visit_Continue(self, node):
        self.test.body.create_keyword(name='CONTINUE', args=node.values,
                                      lineno=node.lineno)